smokey/src/main/java/frc/robot/commands/ArmPlusPickerUptakeCommand.java

```diff
@@ -0,0 +1,287 @@
+// ************************************************************
+// Bishop Blanchet Robotics
+// Home of the Cybears
+// FRC - Charged Up - 2023
+// File: ArmPlusPickerUptakeCommand.java
+// Intent: Forms a command to move the dual part arm with y and z inputs.
+// ************************************************************
+
+// ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ ʕ •ᴥ•ʔ ʕ•ᴥ•  ʔ ʕ  •ᴥ•ʔ ʕ •`ᴥ´•ʔ ʕ° •° ʔ 
+
+package frc.robot.commands;
+
+import java.util.ArrayList;
+
+import edu.wpi.first.wpilibj.PowerDistribution;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.control.ArmPlusPickerAction;
+import frc.robot.subsystems.ArmSubsystem;
+import frc.robot.subsystems.EveryBotPickerSubsystem;
+import frc.robot.subsystems.PowerDistributionPanelWatcherSubsystem;
+
+/*
+ * Forms a command to move the dual part arm and the every bot picker as a unit to help in pickup
+ */
+public class ArmPlusPickerUptakeCommand extends CommandBase {
+
+    private static final double coneInitialSpeed = -0.75;
+    private static final double coneSteadyStateSpeed = -1.0;
+    private static final double coneOverCurrentInitialLimitAmps = 60.0;
+    private static final double coneOverCurrentSteadyStateLimitAmps = 35.5;
+
+    private static final double cubeInitialSpeed = 0.75;
+    private static final double cubeSteadyStateSpeed = 1.0;
+    private static final double cubeOverCurrentInitialLimitAmps = 60.0;
+    private static final double cubeOverCurrentSteadyStateLimitAmps = 35.5;
+
+    private final ArmSubsystem armSubsystem;
+    private final EveryBotPickerSubsystem pickerSubsystem;
+    private final PowerDistributionPanelWatcherSubsystem watcherSubsystem;
+    private boolean isTargetCone = true; 
```

Comment:
would this be better handled by an Enum for game piece type?

Reply:
Likely yes ... good idea